contrib/python-syrupy/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "python-syrupy"
+pkgver = "4.6.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-poetry-core",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest", "python-syrupy"]
```

Comment:
it has itself in checkdepends?

Reply:
never mind that, some leftover before i realized the real issue was missing other dependencies